tests/server-integration/Utils.hs

```diff
@@ -0,0 +1,121 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module Utils
+  ( runServer
+  , cleanEntry
+  , getValueContents
+  , cofferTest
+  , runVault
+  , killVault
+  ) where
+
+import Data.Proxy
+import Data.Text.IO qualified as TIO
+import Network.Wai.Handler.Warp
+import Servant.Server
+
+import Backend.Commands
+import Config
+import Control.Lens
+import Control.Monad (void)
+import Data.Aeson (Value)
+import Data.Aeson.Lens
+import Data.Maybe (fromMaybe)
+import Data.Text (Text)
+import Data.Text qualified as T
+import GHC.IO.Handle (Handle)
+import Network.HTTP.Req
+import System.Environment (lookupEnv)
+import System.Exit
+import System.Process
+  (CreateProcess(std_err, std_out), ProcessHandle, StdStream(NoStream), callProcess, createProcess,
+  getPid, proc)
+import Test.Tasty.HUnit (assertFailure)
+import Toml qualified
+import Web.API
+import Web.Server
+
+readConfig :: FilePath -> IO Config
+readConfig configPath = do
+  text <- TIO.readFile configPath
+  let config = Toml.decode configCodec text
+  case config of
+    Left err -> do
+      TIO.putStrLn $ "An error occurred while decoding '" <> T.pack configPath <> "':"
+      TIO.putStrLn $ Toml.prettyTomlDecodeErrors err
+      exitFailure
+    Right config -> pure config
+
+getConfigPath :: IO FilePath
+getConfigPath = do
+  envConfigPathMb <- lookupEnv "COFFER_CONFIG"
+  pure $ fromMaybe "config.toml" envConfigPathMb
+
+runServer :: IO ()
+runServer = do
```

Comment:
I thought about next thing. `Server integration` task in pipeline already depends on `build` task. So we can run server just like `vault`. What do you think?

Reply:
Hm I think that would make the dev-experience a bit worse, because then you'd have to run `stack install` first before running the tests. Right now, you can just run the tests with no prior setup.

Reply:
Done